pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -262,6 +263,61 @@ func (b *OFBridge) IsConnected() bool {
 	return b.ofSwitch.IsReady()
 }
 
+func (b *OFBridge) AddFlowsInBundle(flows []Flow, modify bool) error {
+	// Create a new transaction.
+	tx := b.ofSwitch.NewTransaction(ofctrl.Atomic)
+	// Open a bundle on the OFSwitch.
+	if err := tx.Begin(); err != nil {
+		return err
+	}
+	// Add Openflow messages into the bundle. "Add" operation is async, the function returns error in constructing and
+	// sending BundleAdd message. It doesn't represent all Openflow entries are added into the bundle even if the function
+	// returns nil. The number of the Openflow entries successfully added into the bundle is returned from function "Complete".
+	installed := false
+	if modify {
+		installed = true
+	}
+	for _, flow := range flows {
+		ofFlow := flow.(*ofFlow)
+		ofFlow.Flow.NextElem = ofFlow.lastAction
+		ofFlow.Flow.UpdateInstallStatus(installed)
```

Comment:
Thanks for the explanation.

More questions:
For new flows, if the status is initialized as false already, we need not to reset the flow status here?
For existing flows, could we consider to set the status to true, when copying the flows?

Reply:
For new flows, it is not necessary to reset the status here.
For existing flows, the status can't be set when copy the flows, it is because the flow actions are set after "copy" in a modification, and it will trigger multiple flowmod messages.

I am thinking of expose an API from ofnet(binding) to allow client(Antrea) to decide the FlowMod operation, then we could remove these status update.